lib/postcssPlugin.js

```diff
@@ -2,18 +2,31 @@
 "use strict"
 const _ = require("lodash")
 const createStylelint = require("./createStylelint")
-const postcss = require("postcss")
+const postcss/*: Object*/ = require("postcss")
 const path = require("path")
 
-module.exports = postcss.plugin("stylelint", function (options) {
+/*:: type OptionsT = {
+  config?: Object;
+  configBasedir?: string;
+  defaultSeverity?: string;
+  from?: string;
+  ignoreDisables?: boolean;
+  ignoreFiles?: string;
+  pluginFunctions?: Object;
+  plugins?: Array<string>;
+  rules?: Object;
+}
+*/
+
+module.exports = postcss.plugin("stylelint", function (options/*: OptionsT*/) {
   options = options || {}
 
   const tailoredOptions/*: Object*/ = options.rules
     ? { config: options }
     : options
-  const stylelint = createStylelint(tailoredOptions)
+  const stylelint/*: stylelint$internalApi*/ = createStylelint(tailoredOptions)
 
-  return (root, result) => {
+  return (root/*: Object*/, result/*: Object*/) => {
```

Comment:
Again, it's not really any help to add `Object` annotations. Let's be more specific.